qiskit/compiler/transpiler.py

```diff
@@ -913,11 +949,17 @@ def _parse_timing_constraints(backend, timing_constraints, num_circuits):
     if backend is None and timing_constraints is None:
         timing_constraints = TimingConstraints()
     else:
-        if timing_constraints is None:
-            # get constraints from backend
-            timing_constraints = getattr(backend.configuration(), "timing_constraints", {})
-        timing_constraints = TimingConstraints(**timing_constraints)
-
+        backend_version = getattr(backend, "version", 0)
+        if not isinstance(backend_version, int):
+            backend_version = 0
+        if backend_version <= 1:
+            if timing_constraints is None:
+                # get constraints from backend
+                timing_constraints = getattr(backend.configuration(), "timing_constraints", {})
+            timing_constraints = TimingConstraints(**timing_constraints)
+        # TODO: Update when BackendV2 has representation of TimingConstraints
```

Comment:
Please make sure this is implemented before actually deployed. Otherwise missing constraints breaks T1 and T2 experiments.

Reply:
Oops, I had forgotten about this, I'll have to add the fields to the target constructor like I did with `dt`. I'll do that now

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/5885/commits/3e7f45b605fa46e05a0d522577b6b3669b6173eb